docs/configure.asciidoc

```diff
@@ -0,0 +1,315 @@
+////
+Goal of this doc:
+Provide a complete reference of all available configuration options and where/how they can be set. (Any Elastic-specific configuration options are listed directly. General OpenTelemetry configuration options are linked.)
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[configure]]
+== Configure
+
+:language: .NET
+:language_lc: dotnet
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+The {distro_name} ("the distro") offers a lot of flexibility...
+
+////
+✅ How users set configuration options
+////
+[discrete]
+[[configure-methods]]
+=== Configuration methods
+
+Configuration of the OpenTelemetry SDK should be performed through the
+mechanisms https://opentelemetry.io/docs/languages/net/automatic/configuration/[documented on the OpenTelemetry website].
+The distro can be further configured using advanced settings when you need complete control of its behavior.
+
+You can set configuration options using a few different methods:
+
+* <<configure-environment-variables,Setting environment variables>>
+* <<configure-iconfiguration-integration,Using the `IConfiguration` integration>>
+* <<configure-manual-configuration,Manually configuring the Elastic distribution>>
+
+[discrete]
+[[configure-environment-variables]]
+==== Environment variables
+
+// ✅ What and why
+The distro can be configured using environment variables.
+This is a cross-platform way to configure the distro and is especially useful in containerized environments.
+
+// ✅ How
+Environment variables are read at startup and can be used to configure the Elastic distribution.
+For details of the various options available and their corresponding environment variable names,
+see <<configure-configuration-options>>.
+
+// Order of operations
+//
+// TO DO: What if a configuration option is set manually in the code?
+// Would that take precedence over environment variables and/or the `IConfiguration` system?
+Environment variables always take precedence over configuration provided by the `IConfiguration`
+system.
```

Comment:
In the docs we mention:

>Environment variables always take precedence over configuration provided by the `IConfiguration`
system.

What if a configuration option is set manually in the code? Would that value take precedence over environment variables and/or the `IConfiguration` system? Or the opposite?

Reply:
Yep!

Code Config > Environment config > IConfiguration